src/Symfony/Component/Lock/Bridge/DynamoDb/README.md

```diff
@@ -0,0 +1,12 @@
+Amazon DynamoDb Lock
+====================
+
+Provides Amazon DynamoDb integration for Symfony Lock.
```

Comment:
```suggestion
Provides Amazon DynamoDB integration for Symfony Lock.
```
Should be spelled like this, only in namespaces or classnames we should use DyanomDb with a lowercased b